.github/workflows/continuous-integration.yml

```diff
@@ -89,13 +90,14 @@ jobs:
         uses: actions/cache@v2
         env:
           cache-name: cache-build-deps
+          sbt-version: 1.5.3
```

Comment:
doh, missed you doing this. nevermind me!

Reply:
FWIW, the `SBT_VERSION` is still in the env here, you shouldn't need to set this here.
```
2021-06-08T19:11:03.7167066Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2021-06-08T19:11:03.7167537Z env:
2021-06-08T19:11:03.7167976Z   JAVA_OPTS: -Dsbt.log.noformat=true
2021-06-08T19:11:03.7168426Z   TRAVIS: true
2021-06-08T19:11:03.7168762Z   SBT_VERSION: 1.5.3
2021-06-08T19:11:03.7169248Z   JAVA_HOME_8.0.292_x64: /opt/hostedtoolcache/jdk/8.0.292/x64
2021-06-08T19:11:03.7169815Z   JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.292/x64
2021-06-08T19:11:03.7170403Z   JAVA_HOME_8_0_292_X64: /opt/hostedtoolcache/jdk/8.0.292/x64
2021-06-08T19:11:03.7170861Z ##[endgroup]
2021-06-08T19:11:03.8505045Z javac 1.8.0_292
2021-06-08T19:11:03.8647208Z ##[group]Run actions/cache@v2
2021-06-08T19:11:03.8647607Z with:
2021-06-08T19:11:03.8648051Z   path: ~/.dodo
~/.ivy2/cache
~/.ivy2/local/com.twitter
~/.sbt

2021-06-08T19:11:03.8648915Z   key: Linux-build-cache-build-deps-1.5.3-2.12.8-1.8
2021-06-08T19:11:03.8649618Z env:
2021-06-08T19:11:03.8650043Z   JAVA_OPTS: -Dsbt.log.noformat=true
2021-06-08T19:11:03.8650481Z   TRAVIS: true
2021-06-08T19:11:03.8650955Z   SBT_VERSION: 1.5.3
2021-06-08T19:11:03.8651421Z   JAVA_HOME_8.0.292_x64: /opt/hostedtoolcache/jdk/8.0.292/x64
2021-06-08T19:11:03.8651999Z   JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.292/x64
2021-06-08T19:11:03.8652576Z   JAVA_HOME_8_0_292_X64: /opt/hostedtoolcache/jdk/8.0.292/x64
2021-06-08T19:11:03.8653125Z   cache-name: cache-build-deps
2021-06-08T19:11:03.8653587Z   sbt-version: 1.5.3
2021-06-08T19:11:03.8653942Z ##[endgroup]
2021-06-08T19:11:04.0452665Z Cache not found for input keys: Linux-build-cache-build-deps-1.5.3-2.12.8-1.8
```

Reply:
I'm just testing things out. I read something that said that I couldn't use a workflow level env variable in a step, but then I found a github issue that said that I could, so I'm trying out different ways. My next version only has the workflow env variable that is used in both jobs. Sorry, this is still in progress!